src/brokers/add-broker/components/AddBrokerForm/BrokerProperties.tsx

```diff
@@ -82,18 +82,10 @@ export const BrokerPropertiesList: FC<BrokerReplicasProp> = ({
 export const BrokerProperties: FC<BrokerIDProp> = ({
   brokerId,
   perBrokerProperties,
-  crName,
-  targetNs,
 }) => {
-  console.log('configuring broker ', crName, 'in namespace', targetNs);
-
   const [currentConfigItem, setCurrentConfigItem] = useState('');
 
-  const onSelectBrokerConfigItem = (
-    selectedItem: any,
-    selectedItemProps: any,
-  ) => {
-    console.log('new selection selected', selectedItem);
+  const onSelectBrokerConfigItem = (selectedItemProps: any) => {
```

Comment:
Here by removing the first parameter of the function it made `selectedItemProps` be the former first parameter instead. What should be done here:

```typescript
const onSelectBrokerConfigItem = (
  _selectedItem: any,
  selectedItemProps: any,
) => {
  setCurrentConfigItem(selectedItemProps.itemId);
};

```